src/model/class.ts

```diff
@@ -122,8 +121,8 @@ export class Class implements Feature {
   }
   readonly jsdoc: jsdocLib.Annotation|undefined;
   description: string;
-  readonly properties: Property[] = [];
-  readonly methods: Method[] = [];
+  readonly properties = new Map<string, Property>();
+  readonly methods: Map<string, Method> = new Map();
```

Comment:
no reason. standardized on putting the arguments on the constructor, as there are semi-rare cases where that does the right thing better